.github/workflows/ci.yml

```diff
@@ -147,7 +146,7 @@ jobs:
         timeout-minutes: 20
         run: |
           set -euo pipefail
-          go test -json -v $(go list ./... | grep -Ev '/blockchain|/badgerTrie|/consensus/dman|/testutils') 2>&1 | tee /tmp/gotest.log | gotestfmt
+          go test -json -v ./... 2>&1 | tee /tmp/gotest.log | gotestfmt
```

Comment:
There are some tests failing on CI: 
https://github.com/MadBase/MadNet/runs/6577276432?check_suite_focus=true
Maybe we will need to add the flake tags to them?

Reply:
Good call! Just refreshed the merge branch, if they recur I'll get them added.